playbooks/tasks/templates/prometheus/prometheus.yml.j2

```diff
@@ -18,12 +18,15 @@ remote_write:
       max_shards: 100
     url: {{ push_url }}
     remote_timeout: 10s
+    honor_labels: true 
```

Comment:
honor_labels should only be necessary if there's a conflict of labels. Which labels are conflicting?